chromium_src/chrome/browser/permissions/chrome_permissions_client.cc

```diff
@@ -39,7 +40,9 @@ bool ChromePermissionsClient::BraveCanBypassEmbeddingOriginCheck(
           permissions::ContentSettingsTypeToRequestType(type),
           url::Origin::Create(requesting_origin), &original_requesting_origin,
           nullptr) &&
-      original_requesting_origin == url::Origin::Create(embedding_origin)) {
+      net::registry_controlled_domains::SameDomainOrHost(
+          original_requesting_origin, url::Origin::Create(embedding_origin),
+          net::registry_controlled_domains::INCLUDE_PRIVATE_REGISTRIES)) {
```

Comment:
just wanted to double check - `embedding_origin` is the top level (main frame) origin as opposed to the parent origin? ex: if a.com embeds b.com which embeds c.com and the requesting origin is c.com, embedding origin is a.com not b.com.